### DISCUSSION TITLE
How do you add a https clickhouse connection with authorisaiton headers?

### DISCUSSION THREAD

**Discussion by ShravanSunder** [grafana/clickhouse-datasource#625] [Q&A]
I wanted to use this with grafana cloud to view data in cloudflare analytics engine that uses clickhouse.  https://developers.cloudflare.com/analytics/analytics-engine/grafana/

They need use to use authorisation header with https.  How can this be done in this plug in?

If not can we request this feature?

**Comment by adamyeats** [comment]
@ShravanSunder Thanks for your feedback here! This is currently not possible in the plugin, however it does look to be quite straightforward to implement on our side. I will add this to our backlog as a feature request and we'll take a look as soon as we can prioritise it.

**Comment by adamyeats** [comment]
Added an issue for this here: https://github.com/grafana/clickhouse-datasource/issues/626

**Comment by SpencerTorres** [comment]
This feature is now available in [v4.x](https://github.com/grafana/clickhouse-datasource/releases/tag/v4.0.1)
